src/Compilers/CSharp/Test/Emit3/ExtensionTypeTests.cs

```diff
@@ -35273,40 +35251,29 @@
     {
         var source = """
 // we'll lookup here
+int.StaticMethod();
+
 public implicit extension E<T> for T where T : struct
 {
     public static int StaticField = 0;
     public static int StaticProperty => 0;
-    public static void StaticMethod() { }
+    public static void StaticMethod() { System.Console.Write("ran"); }
     public class Nested { }
     public static event System.Action StaticEvent { add { } remove { } }
 }
 """;
         var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
-        // PROTOTYPE should warn about hiding
-        comp.VerifyDiagnostics();
+        CompileAndVerify(comp, expectedOutput: IncludeExpectedOutput("ran"), verify: Verification.FailsPEVerify).VerifyDiagnostics();
 
         var tree = comp.SyntaxTrees.First();
         var model = comp.GetSemanticModel(tree);
 
         var o = ((Compilation)comp).GetSpecialType(SpecialType.System_Object);
         AssertSetStrictlyEqual(objectSymbols, model.LookupSymbols(position: 0, o).ToTestDisplayStrings());
-
-        string[] eStaticSymbols = [
-            .. objectStaticSymbols,
-            "System.Int32 E<System.Object>.StaticProperty { get; }",
-            "void E<System.Object>.StaticMethod()",
-            "E<System.Object>.Nested",
-            "event System.Action E<System.Object>.StaticEvent",
-            "System.Int32 E<System.Object>.StaticField"];
-
-        // PROTOTYPE(instance) We'll want LookupSymbols to return extension type members too
         AssertSetStrictlyEqual(objectSymbols, model.LookupSymbols(position: 0, o, includeReducedExtensionMethods: true).ToTestDisplayStrings());
 
-        AssertSetStrictlyEqual(eStaticSymbols, model.LookupStaticMembers(position: 0, o).ToTestDisplayStrings());
-
-        AssertSetStrictlyEqual(["E<System.Object>.Nested"], model.LookupNamespacesAndTypes(position: 0, o).ToTestDisplayStrings());
-        AssertSetStrictlyEqual(eStaticSymbols, model.LookupStaticMembers(position: 0, o).ToTestDisplayStrings());
+        AssertSetStrictlyEqual([], model.LookupNamespacesAndTypes(position: 0, o).ToTestDisplayStrings());
+        AssertSetStrictlyEqual(objectStaticSymbols, model.LookupStaticMembers(position: 0, o).ToTestDisplayStrings());
```

Comment:
>AssertSetStrictlyEqual(objectStaticSymbols, model.LookupStaticMembers(position: 0, o).ToTestDisplayStrings()); [](http://example.com/codeflow?start=8&length=110)

It is not obvious that this change is expected.

Reply:
The extension members are not compatible with the type `object` because of the `struct` constraint.